tests/e2e/tests/critical-path/monitor/monitor.e2e.ts

```diff
@@ -0,0 +1,83 @@
+        await t.maximizeWindow();
+        await userAgreementPage.acceptLicenseTerms();
+        await t.expect(addRedisDatabasePage.addDatabaseButton.exists).ok('The add redis database view', { timeout: 20000 });
+        await addNewStandaloneDatabase(ossStandaloneConfig);
+        await myRedisDatabasePage.clickOnDBByName(ossStandaloneConfig.databaseName);
+    })
+test('Verify that user can work with Monitor', async t => {
+    const command = 'set';
+    //Expand Monitor panel
+    await t.click(monitorPage.expandMonitor);
+    //Check that monitor is opened
+    await t.expect(monitorPage.monitorArea.exists).ok('Monitor area');
+    await t.expect(monitorPage.startMonitorButton.exists).ok('Start monitor button');
+    //Verify that user can see warning message about monitor
+    await t.expect(monitorPage.monitorWarningMessage.exists).ok('Monitor warning message');
+    await monitorPage.monitorWarningMessage.withText('Running Monitor will decrease throughput, avoid running it in production databases');
+    //Verify that user can run monitor
+    await t.click(monitorPage.startMonitorButton);
+    await t.expect(monitorPage.monitorIsStartedText.exists).ok('"Monitor is started" text');
+    //Verify that user can see run commands in monitor
+    const keyName = `${getRandomKeyName(10)}-key`;
+    const keyValue = `${getRandomKeyName(10)}-value`;
+    await cliPage.getSuccessCommandResultFromCli(`${command} ${keyName} ${keyValue}`);
+    // await monitorPage.checkLineExistenceInMonitor(command);
+    await monitorPage.checkCommandInMonitorResults(command, [keyName, keyValue]);
+});
+test('Verify that user can see the list of all commands from all clients ran for this Redis database in the list of results in Monitor', async t => {
+    //Define commands in different clients
+    const cli_command = 'command';
+    const workbench_command = 'hello';
+    const common_command = 'info';
+    const browser_command = 'ttl';
+    //Expand Monitor panel
+    await t.click(monitorPage.expandMonitor);
+    //Start monitor (using run button in header)
+    await t.click(monitorPage.runMonitorToggle);
+    //Send command in CLI
+    await cliPage.getSuccessCommandResultFromCli(cli_command);
+    //Check that command from CLI is displayed in monitor
+    await monitorPage.checkCommandInMonitorResults(cli_command);
+    //Refresh the page to send command from Browser client
+    await t.click(browserPage.refreshKeysButton);
+    //Check the command from browser client
+    await monitorPage.checkCommandInMonitorResults(browser_command);
+    //Open Workbench page to create new client
+    await t.click(myRedisDatabasePage.workbenchButton);
+    //Send command in Workbench
+    await workbenchPage.sendCommandInWorkbench(workbench_command);
+    //Check that command from Workbench is displayed in monitor
+    await monitorPage.checkCommandInMonitorResults(workbench_command);
+    //Wait for info request
+    await t.wait(5000);
```

Comment:
Why this, and not `{ timeout: 5000 }` within the function when you assert it?

Reply:
Fixed